bundler.d/ovirt.rb

```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'fog-ovirt', '~> 1.1.2'
+  gem 'fog-ovirt', '~> 1.1.5'
```

Comment:
@theforeman/packaging mind acking?

Reply:
I did not see a PR for it so I opened: https://github.com/theforeman/foreman-packaging/pull/3433